<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>256</width>
      <height>641</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>259</x>
      <y>-1</y>
      <width>791</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="asdfasdfa">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="name">
       <property name="text">
        <string>View Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="addressLabel">
       <property name="text">
        <string>Address:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="addressLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="widthLabel">
       <property name="text">
        <string>Width:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="widthLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="heightLabel">
       <property name="text">
        <string>Height:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="heightLineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="resizeableLabel">
       <property name="text">
        <string>Resizeable:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="resizeableCheckBox">
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="fullescreenLabel">
       <property name="text">
        <string>Fullscreen:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="fullescreenCheckBox"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="persistantCookiesLabel">
       <property name="text">
        <string>Persistant Cookies:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QCheckBox" name="persistantCookiesCheckBox">
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="enableCachingLabel">
       <property name="text">
        <string>Enable Caching:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QCheckBox" name="enableCachingCheckBox">
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="maxCacheSizeLabel">
       <property name="text">
        <string>Max Cache Size (MB): </string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="maxCacheSizeLineEdit">
       <property name="text">
        <string>500</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="enableProxyLabel">
       <property name="text">
        <string>Enable Proxy:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QCheckBox" name="enableProxyCheckBox"/>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="proxyHostLabel">
       <property name="text">
        <string>Proxy Host:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="proxyHostLineEdit"/>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="proxyPortLabel">
       <property name="text">
        <string>Proxy Port:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLineEdit" name="proxyPortLineEdit"/>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="groupLabel">
       <property name="text">
        <string>Group</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QComboBox" name="groupComboBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="updateSettingsButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>580</y>
      <width>761</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Settings</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1048</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionImport"/>
    <addaction name="separator"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionBackup"/>
    <addaction name="actionCopy_Filesystem"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
    <addaction name="actionRefresh_Lists"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionCreate_Launcher_Shortcut"/>
    <addaction name="actionRemove"/>
    <addaction name="actionDuplicate"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionWebView_Website"/>
    <addaction name="actionView_Logs"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionToggle_Fullscreen"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionWebView_Website">
   <property name="text">
    <string>WebView Website</string>
   </property>
  </action>
  <action name="actionView_Logs">
   <property name="text">
    <string>View Logs</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionRefresh_Lists">
   <property name="text">
    <string>Refresh Lists</string>
   </property>
  </action>
  <action name="actionCopy_Filesystem">
   <property name="text">
    <string>Copy Filesystem</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionBackup">
   <property name="text">
    <string>Backup</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionCreate_Launcher_Shortcut">
   <property name="text">
    <string>Create Launcher/Shortcut</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionDuplicate">
   <property name="text">
    <string>Duplicate</string>
   </property>
  </action>
  <action name="actionNew_2">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionRemove_2">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionDuplicate_2">
   <property name="text">
    <string>Duplicate</string>
   </property>
  </action>
  <action name="actionToggle_Fullscreen">
   <property name="text">
    <string>Toggle Fullscreen</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
